### DISCUSSION TITLE
Profile Improvements

### DISCUSSION THREAD

**Discussion by WASasquatch** [civitai/civitai#329] [Ideas]
I'm sure much of this has been talked about in bits and pieces around the ideas sections, I just wanted to highlight some ideas here in no particular order.

- Highlight Content: Highlight models or other creations for users to view on your profile as featured content.
- Timeline Discussions: Talk about upcoming creations you are doing share preview images, get feedback, or ideas, etc
- Verified Uploader: Their content has been vetted as consistently of good qualtiy *(could be automated via heart ratio -- using reviews doesn't seem like best option as even on popular models, reviews are hardly given over likes/downloads)*
- Donate link: A link type for donation gateways, with it's own icon to indicate it as such

Off-Topic: 
- Partners / Credit: Select other users as a partner on a release, whether they trained for you, gathered resources, etc, etc. Maybe allow a small field for what their credit is about (training, resources, advice, etc)

**Comment by obentouso** [comment]
Profile introduction would be good too, to be able understand what kind of direction in term of style or certain type of character the creator will be going or preferred in making the models, checkpoint, etc. And more achievement badge or milestones badge for new creators

**Reply by WASasquatch** [reply]
Good point, some sort of small bio blurb or something to let people know what they're all about. I know I've gotten some requests for things way outside my interest areas haha
